pkg/test/framework/components/echo/flags.go

```diff
@@ -23,7 +23,7 @@ import (
 
 var (
 	callTimeout = 20 * time.Second
-	callDelay   = 100 * time.Millisecond
+	callDelay   = 200 * time.Millisecond
```

Comment:
just curious about the motivation here?

Reply:
Just thought 100ms might be small in general. I originally picked `100` since that's what was used by multicluster. However, as I go through more tests, > `200` seems more common. I can revert for now if you'd prefer.